src/hw_isolation_event/hw_status_manager.cpp

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#include "config.h"
+
+#include "hw_isolation_event/hw_status_manager.hpp"
+
```

Comment:
why 1 line gap?

Reply:
I think it will be added by the clang to differentiate the local repo file vs standard and other external dependent component header files.

I have run the clang for each commit but, I will confirm this by looking at it one more time.